### PR TITLE
updated v2 imports

### PR DESCRIPTION
**Pull request by sadMaxim**

### CODE REVIEW

File: src/BotPlutusInterface/CardanoCLI.hs [hunk 1]
```diff
@@ -70,7 +70,8 @@ import Ledger.Interval (
   UpperBound (UpperBound),
  )
 import Ledger.Scripts (Datum, DatumHash (..))
-import Ledger.Scripts qualified as Scripts
+--import Ledger.Scripts qualified as Scripts
```
Comment (gege251) [inline src/BotPlutusInterface/CardanoCLI.hs:73]: We can remove the commented line

File: src/BotPlutusInterface/Files.hs [hunk 2]
```diff
@@ -295,7 +295,7 @@ mkDummyPrivateKey (PubKey (LedgerBytes pubkey)) =
         Crypto.xprv $
           mconcat [dummyPrivKey, dummyPrivKeySuffix, pubkeyBS, dummyChainCode]
 
-serialiseScript :: Script -> PlutusScript PlutusScriptV1
+serialiseScript :: Script -> PlutusScript PlutusScriptV2
```
Comment (mikekeke) [inline src/BotPlutusInterface/Files.hs:298]: Will V1 scripts keep working with this change?

Comment (sadMaxim) [reply src/BotPlutusInterface/Files.hs:298]: No it won't work with V1. Core things is difference in V1/V2 ScriptContext and validatorHash

**Comment by mikekeke** [comment]
Changes from this PR migrated to PR for newer revision: #151

**Pull request closed without merging**